src/api/java/com/minecolonies/api/colony/buildings/IBuildingWorker.java

```diff
@@ -207,6 +207,20 @@
     @NotNull
     Skill getSecondarySkill();
 
+    /**
+     * Crafting Speed skill
+     * @return the crafting speed skill
+     */
+    @NotNull
+    Skill getCraftSpeedSkill();
+
+    /**
+     *  Recipe Improvement skill getter
+     * @return the recipe improvement skill
+     */
+    @NotNull
+    Skill getRecipeImprovementSkill();
+
```

Comment:
this should probably go into the crafter building

Reply:
done.